### PR TITLE
feat(wds-mcp): mcp server 패키지 추가

### PR DESCRIPTION
**Pull request by Sh031224**
## Summary by CodeRabbit

# 릴리스 노트

* **New Features**
  * WDS MCP 패키지 추가: 문서·유틸리티 검색/조회 서버 및 CLI 진입점 포함

* **Documentation**
  * 타이포그래피 문서에 Variants(변형) 섹션 및 사용 예시 추가
  * WDS 및 wds-mcp용 상세 README 추가

* **Configuration**
  * 기본 언어를 한국어(ko-KR)로 변경
  * 시(포엠) 출력 비활성화 및 유닛테스트 생성/검토 설정 활성화

* **Dependencies**
  * 문서용 라이브러리 버전 업데이트

* **Style**
  * 코드 블록 색상 스타일 개선

### CODE REVIEW

File: .coderabbit.yaml [hunk 1]
```diff
@@ -18,3 +21,65 @@ reviews:
...
+
+            it('should pass accessibility tests', async () => {
+              const { rerender } = render(<DateCalendar {...defaultProps} />);
+              expect(await axe(screen.getByRole('rowgroup'))).toHaveNoViolations();
+              // Additional accessibility assertions...
+            });
+          });
+          ```
+
+          - Be sure to include accessibility and edge case tests where applicable.
```
[the remark this answers is not in the document]
Comment (Sh031224) [reply .coderabbit.yaml:85]: 수정했어

File: packages/wds-mcp/src/server.ts [hunk 2]
```diff
@@ -0,0 +1,493 @@
...
+  color: \${theme.semantic.label.normal};
+\`;
+\`\`\`
+
+or with the Typography component like:
+
+\`\`\`tsx
+<Typography color="semantic.label.normal" />
+\`\`\`
+`,
```
[the remark this answers is not in the document]
Comment (Sh031224) [reply packages/wds-mcp/src/server.ts:367]: 수정했어

File: packages/wds-mcp/src/helpers/index.ts [hunk 3]
```diff
@@ -0,0 +1,167 @@
+import * as icons from '@wanteddev/wds-icon';
+import { theme } from '@wanteddev/wds-theme';
+import { load } from 'cheerio';
+import { camelCase, kebabCase } from 'change-case';
+
+import api from '../../../../docs/generated/api.json' assert { type: 'json' };
```
[the remark this answers is not in the document]
Comment (Sh031224) [reply packages/wds-mcp/src/helpers/index.ts]: 수정했어

File: packages/wds-mcp/src/server.ts [hunk 4]
```diff
@@ -0,0 +1,493 @@
...
+    }),
+  },
+  async ({ functionName }) => {
+    const fetchUrl = await getUtilityFunctionUrl(functionName);
+
+    if (!fetchUrl) {
+      return {
+        content: [],
+      };
+    }
```
[the remark this answers is not in the document]
Comment (Sh031224) [reply packages/wds-mcp/src/server.ts:465]: 수정했어

File: packages/wds-mcp/src/server.ts [hunk 5]
```diff
@@ -0,0 +1,493 @@
...
+
+    const fetchUrl = await getComponentUrl(match.name);
+
+    if (!fetchUrl) {
+      return {
+        content: [],
+      };
+    }
```
[the remark this answers is not in the document]
Comment (Sh031224) [reply packages/wds-mcp/src/server.ts:243]: 반영했어

File: packages/wds-mcp/src/index.ts [hunk 6]
```diff
@@ -0,0 +1,7 @@
+import { StdioServerTransport } from '@modelcontextprotocol/sdk/server/stdio.js';
+
+import { server } from './server';
+
+const transport = new StdioServerTransport();
+
+await server.connect(transport);
```
[the remark this answers is not in the document]
Comment (Sh031224) [reply packages/wds-mcp/src/index.ts:7]: 이건 괜찮을듯 어차피 mcp 서버라

File: packages/wds-mcp/bin/wds-mcp [hunk 7]
```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env node
+// pnpm requires that package.json bin entries refer to files that exist prior
+// to running any lifecycle scripts. So here we have a stub entry that requires
+// the transpiled real binary.
+import('../dist/index.mjs');
```
[the remark this answers is not in the document]
Comment (Sh031224) [reply packages/wds-mcp/bin/wds-mcp:5]: 이것도 괜찮을듯 import가 실패나면 빌드가 제대로 안된건데 그럼 빌드에서 오류났겠지
